docs/gsc-disable-initial-evaluation-spec.md

```
@@ -0,0 +1,121 @@
+
+## 背景・課題
+
+コンテンツ（記事）をGSC評価に登録した際、初回の評価実行時に「前回の順位」が存在しないにもかかわらず改善提案が生成される。100記事を登録すると100件の改善提案が出るが、比較対象がないため評価としての意義がない。
+
+初回は「基準値（ベースライン）の記録」であり、「次回にどう変化したか」が本来の評価。
+
+## 現状の動作（変更前）
+
+### 初回評価の処理フロー
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)